astacus/common/rohmustorage.py

```diff
@@ -109,27 +113,38 @@ def __init__(self, config: RohmuConfig, *, storage: str | None = None) -> None:
 
     @rohmu_error_wrapper
     def _download_key_to_file(self, key, f: BinaryIO) -> bool:
-        raw_metadata: dict = self.storage.get_metadata_for_key(key)
         with tempfile.TemporaryFile(dir=self.config.temporary_directory) as temp_file:
+            raw_metadata: Metadata = self.storage.get_metadata_for_key(key)
```

Comment:
The commit description says this call was removed, but it's just moved.

We should be using the return value of `get_contents_to_fileobj`.